ambari-server/src/main/python/ambari-server.py

```diff
@@ -490,7 +490,7 @@ def init_setup_parser_options(parser):
 
   other_group.add_option('-j', '--java-home', default=None,
                          help="Use specified java_home.  Must be valid on all hosts")
-  other_group.add_option('--ambari-java-home',dest="ambari_java_home", default=None,
+  other_group.add_option('--ambari-java-home',dest="ambari_java_home",
```

Comment:
This change has already been merged in another pull request. A new PR should be created based on a clean community branch as your working branch.